LICENSE

```diff
@@ -0,0 +1,13 @@
+   Copyright 2019 PLUS3 IT SYSTEMS
```

Comment:
Please use the full license, e.g. https://github.com/plus3it/terraform-aws-tardigrade-cloudtrail/blob/master/LICENSE. And be sure to change the copyright line... https://github.com/plus3it/terraform-aws-tardigrade-cloudtrail/blob/master/LICENSE#L189 (Copyright 2019 Maintainers of plus3it/vault-formula)